pyds9.py

```diff
@@ -1,14 +1,20 @@
+from __future__ import print_function
```

Comment:
My recommendation would be that you use
```
from __future__ import absolute_import, division, print_function, unicode_literals
```
i.e. make Python 2 behave as much as Python 3.
This is in the [Astropy coding guidelines](http://astropy.readthedocs.org/en/latest/development/codeguide.html#interface-and-dependencies) and many other packages that need to support Python 2 / 3 do it this way. (You're already using six where it's not easy to write the code so that it works on Python 2 / 3 directly, that's good.)